pom.xml

```diff
@@ -73,6 +73,8 @@
         <skin.groupId>org.apache.maven.skins</skin.groupId>
         <skin.artifactId>maven-fluido-skin</skin.artifactId>
         <skin.version>1.12.0</skin.version>
+        <!--        XXX SHOULD BE RELEASE 9.0 and superior artefacts  changes to Apache Licence -->
```

Comment:
this comment is probably no longer important. Nobody would set the version so far back.

Reply:
clenanup and forcepushed